src/test/kotlin/net/maizegenetics/phgv2/cli/CreateRefVCFTest.kt

```diff
@@ -151,5 +150,27 @@ class BuildRefVCFTest {
         val numberOfReferenceLines = bufferedReader(outFileCompressed).readLines().filter { it.startsWith("##reference") }.size
         assertEquals(1, numberOfReferenceLines)
 
+        // Verify the RefAllele for the first haplotype for each chromosome is correct
+        val genomeLines = bufferedReader(genome).readLines()
+        val firstChrom = genomeLines.filter { it.startsWith(">") }[0].removePrefix(">")
+        val secondChrom = genomeLines.filter { it.startsWith(">") }[1].removePrefix(">")
+
+        // get the vcf data lines
+        val vcfDataLines = bufferedReader(outFileCompressed).readLines().filter { !it.startsWith("#") }
```

Comment:
Will this close the file? Do you need .use{}?

Reply:
I think it is fine.